README.md

```diff
@@ -1,6 +1,121 @@
-# Dr. Shub, MD
-## Also known as DrsHub
+# Dr. Shub, MD (Also known as DrsHub)
 
-## Logging
+## Overview
+DrsHub is a Java Spring Boot rewrite of the Cloud Function [Martha](https://github.com/broadinstitute/martha), specifically, its v3 API.
+The purpose of DrsHub is to resolve [DRS](https://ga4gh.github.io/data-repository-service-schemas/preview/develop/docs/) URLs for Terra services.
+
+## DRS Providers
+Currently, DrsHub supports the following DRS Providers:
+
+- Terra Data Repo (TDR)
+  - Authentication Types:
+    - Bearer Token
+- NHGRI Analysis Visualization and Informatics Lab-space (The AnVIL)
+  - Authentication Types:
+    - Fence Token
+- BioData Catalyst (BDC)
+  - Authentication Types:
+    - Fence Token
+- NCI Cancer Research / Proteomics Data Commons (CRDC / PDC)
+  - Authentication Types:
+    - Fence Token
+- Gabriella Miller Kids First DRC
+  - Authentication Types:
+    - Fence Token
+- Passport Test Provider
+  - Authentication Types:
+    - Passports
+    - Fence Tokens
+
+## Usage
+To resolve a DRS URL, perform an HTTP `POST` to the `/api/v4/drs/resolve`.
+The content-type of your request should be application/json with the content/body of your request encoded accordingly.
+
+Request bodies should look like 
+```json
+{
+  "url": "string",
+  "fields": ["string"]
+}
+```
+where `url` is the DRS URL to resolve and `fields` is any of
+```text
+accessUrl
+bondProvider
+bucket
+contentType
+fileName
+googleServiceAccount
+gsUri
+hashes
+localizationPath
+name
+size
+timeCreated
+timeUpdated
+```
+
+By default, the response will include 
+```text
+bucket
+contentType
+fileName
+gsUri
+hashes
+localizationPath
+name
+size
+timeCreated
+timeUpdated
+googleServiceAccount
+```
+but this can be changed by providing values in the `fields` array.
+
+## Architecture
+DrsHub is a Java 17 Spring Boot application running in Kubernetes. As it simply resolves urls and doesn't have any state, it has no database. For developer convenience, a Swagger UI is provided.
```

Comment:
Can we provide the path to the Swagger UI?

Reply:
I think we would make `DrsHub` a link to the Prod swagger, once it exists and is deployed.